server/src/user-org/dto/create-user-org.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { Organization } from '../../organizations/entities/organization.entity';
+import { User } from '../../users/entities/user.entity';
+import { ApprovalStatus, Role } from '../constants';
+
+export class CreateUserOrganizationDto {
+  approvalStatus: ApprovalStatus;
+  organization: Organization;
+  role: Role;
+  user: User;
```

Comment:
Are any of these required? we could use the dto validator library

Reply:
good call, added some of the validators that we use elsewhere, where appropriate
these are required by default right?